shopify_app.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("jwt", ">= 2.2.3")
   s.add_runtime_dependency("rails", "> 5.2.1")
   s.add_runtime_dependency("redirect_safely", "~> 1.0")
-  s.add_runtime_dependency("shopify_api", "~> 11.0")
+  s.add_runtime_dependency("shopify_api", "~> 11.1.0")
```

I think we shouldn't be locking down the minor version here, right?

```suggestion
  s.add_runtime_dependency("shopify_api", "~> 11.1")
```